client/src/components/Map/index.tsx

```diff
@@ -80,18 +123,73 @@ function MapLi({
   );
 }
 
-function Map({ forLanding = false }: MapProps): React.ReactElement {
-  return (
-    <div className='map-ui' data-test-label='curriculum-map'>
-      <ul>
-        {flatSuperBlockMap.map((superBlock, i) => (
-          <MapLi key={i} superBlock={superBlock} landing={forLanding} />
-        ))}
-      </ul>
-    </div>
-  );
+function Map({
+  forLanding = false,
+  isSignedIn,
+  currentCerts
+}: MapProps): React.ReactElement {
+  const certSlugTypeMapTyped: { [key: string]: string } = certSlugTypeMap;
+  const superBlockCertTypeMapTyped: { [key: string]: string } =
+    superBlockCertTypeMap;
+
+  return (() => {
+    let startingIndex = 0;
+    const superBlockMaps = (
+      <div className='map-ui' data-test-label='curriculum-map'>
+        <ul>
```

Comment:
```suggestion
        <ul>
```

This should be an `ol` since order is important in these lists.